ios/Implementation.swift

```diff
@@ -146,6 +148,41 @@ public class Implementation: NSObject {
     }
 }
 
+extension Implementation {
+    private static var pushToken: Data?
+    private static var notificationResponse: UNNotificationResponse?
+
+    @objc
+    public static func setPushToken(_ deviceToken: Data?) {
+        guard let impl = Implementation.implementation else {
+            pushToken = deviceToken
+            return
+        }
+
+        impl.setPushToken(deviceToken)
+    }
+
+    @objc
+    public static func didReceiveNotification(response: UNNotificationResponse, completionHandler: @escaping () -> Void) -> Bool {
+        guard let impl = Implementation.implementation else {
+            Implementation.notificationResponse = response
```

Comment:
just a code style nitpick - this one uses `Implementation.` static name qualification, L158 above does not. should they be consistent?